UnitySDK/Assets/validate_meta_files.py

```diff
@@ -0,0 +1,42 @@
+import json
+import os
+
+
+def main():
+    asset_path = "UnitySDK/Assets"
```

Comment:
This assumes that you are always running in the root path of the repo, which I guess is fine since you will only run it in the circleci.

Reply:
Yeah, I guess we could add the path as a parameter if we need to later. But I think it's OK to leave hardcoded for now.